ablySpec/RealtimeClient.connection.swift

```diff
@@ -328,6 +328,48 @@ class RealtimeClientConnection: QuickSpec {
                     expect(errorInfo).toNot(beNil())
                 }
             }
+
+            class TotalReach {
+                // Easy way to create an atomic var
+                static var shared = 0
+                // This prevents others from using the default '()' initializer
+                private init() {}
+            }
+
+            // RTN5
+            it("basic operations should work simultaneously") {
+                let options = AblyTests.commonAppSetup()
+                options.echoMessages = false
+                var disposable = [ARTRealtime]()
+                let max = 50
+                let channelName = "chat"
+                TotalReach.shared = 0
+
+                for _ in 1...max {
+                    disposable.append(ARTRealtime(options: options))
+
+                    let channel = disposable.last?.channel(channelName)
```

Comment:
instead of using last?, why not simply declare the client to make the code more readable such as:

    let client = ARTRealtime(options: options)
    disposable.append(client)
    let channel = client.channel(channelName)